netci.groovy

```diff
@@ -831,9 +835,17 @@ combinedScenarios.each { scenario ->
                                         // Get corefx
                                         buildCommands += "git clone https://github.com/dotnet/corefx fx"
                                         
+                                        // Set environment variable
+                                        def envVars = Constants.jitStressModeScenarios[scenario]
+                                        envVars.each{ VarName, Value   ->
+                                            if (VarName != '') {
+                                                buildCommands += "export ${VarName}=${Value}"
```

Comment:
Are you sure this will work?  Each buildCommand element of buildCommands is a separate bash invocation so I don't think these will get set.  Can you also just use the code that was generating environment variable scripts already?